### ISSUE TITLE
excluded_topics parameter should encode regexp before passing it to CC

### ISSUE THREAD

**Issue by efeg** [linkedin/cruise-control-ui#40]
Excerpts copied from the issue: https://github.com/linkedin/cruise-control/issues/979

--

For the following rebalance request:
```
         https://cruise-control.company.net/kafkacruisecontrol/rebalance?dryrun=true
		&goals=RackAwareGoal
		&skip_hard_goal_check=true
		&excluded_topics=(_heartbeats|__CruiseControlMetrics)
		&json=true
```

Running from GUI get the regexp wrong (excluded_topics=_heartbeats%257C__CruiseControlMetrics)
DEBUG:
```
Oct 25 08:34:37 cruise-control-app.company.net cruise-control[16209]: [2019-10-25 08:34:37,032] DEBUG REQUEST POST /kafkacruisecontrol/rebalance on HttpChannelOverHttp@17e809fb{r=1,c=false,c=false/false,a=DISPATCHED,uri=//localhost:8081/kafkacruisecontrol/rebalance?dryrun=true&goals=RackAwareGoal&skip_hard_goal_check=true&excluded_topics=_heartbeats%257C__CruiseControlMetrics&json=true,age=0} (org.eclipse.jetty.server.Server)
```
Running via curl works:
DEBUG:
```
Oct 25 08:36:30 cruise-control-app.company.net cruise-control[16209]: [2019-10-25 08:36:30,043] DEBUG REQUEST POST /kafkacruisecontrol/rebalance on HttpChannelOverHttp@75afb24c{r=1,c=false,c=false/false,a=DISPATCHED,uri=//localhost:8081/kafkacruisecontrol/rebalance?dryrun=true&excluded_topics=_heartbeats|__CruiseControlMetrics&goals=RackAwareGoal&skip_hard_goal_check=true&json=true,age=0} (org.eclipse.jetty.server.Server)
```
It drills-down to `%257C` that will be translated to `%|` for the gui call which don't work

--

Sending just a %7C instead of the pipe | works as well. But as stated a %257C dont.

**Comment by nareshv** [comment]
data is being passed to xss-filtering due to which there seem to be extra characters.

**Comment by nareshv** [comment]
![cc-url-encode-fix-bug40](https://user-images.githubusercontent.com/1161803/70030055-67f86600-15ce-11ea-8a5c-28543bad5c41.png)

v0.3.2 will have this fix

**Comment by nareshv** [comment]
released v0.3.2 of ccui
